package.json

```diff
@@ -19,7 +19,7 @@
     "electron-is-dev": "^2.0.0",
     "react": "^18.1.0",
     "react-dom": "^18.1.0",
-    "react-scripts": "4.0.3",
```

Comment:
We had to downgrade `react-scripts` to `4.0.3` because many libraries don't work with Webpack v5. Please revert the version change.